modules/ROOT/pages/lb-cert-validation.adoc

```diff
@@ -45,17 +45,16 @@ You can send a `PATCH` request to the `/organizations/{orgid}/vpcs/{vpcId}/loadb
 
 [NOTE]
 --
-We recommend you use the CloudHub REST API to programmatically update your revocation lists. +
-In order to get the necessary `vpcId`, and loadbalancerId from the CLI, you can use a xref:anypoint-platform-cli.adoc#cloudhub-vpc-describe-json[`vpc JSON describe`] and xref:anypoint-platform-cli.adoc#cloudhub-load-balancer-describe-json[`load-balancer JSON describe`] command respectively.
+We recommend that you use the CloudHub REST API to programmatically update your revocation lists. +
```

Comment:
Is it really necessary for this to be boxed as a Note, right after the syntax example? Readers will probably pay just as much attention to plain text.

Reply:
Avoid referring to "we" or "MuleSoft recommends." How about "You should use," "Use," or "Best practice is to use"?